lib/core/src/base64.cpp

```diff
@@ -4,135 +4,139 @@
 
 #include "irods/rodsClient.h"
 #include "irods/base64.h"
```

Comment:
Please swap the order of these includes and delete all lines above them.

Reply:
done

Reply:
done